styleguide/derek/pattern-components/webinar-item/_webinar-item.scss

```diff
@@ -44,12 +44,22 @@
   font-weight: $font-weight-regular;
 }
 
-.rsWebinarTopic-item-link a {
-  color: $rackspace-red;
-  font-weight: $font-weight-regular;
-  text-decoration: underline;
+a {
+  .rsWebinarTopic-item-link {
+    color: $rackspace-red;
+    font-weight: $font-weight-regular;
+    text-decoration: underline;
+  }
```

Comment:
What is this change for? Are we using this anywhere in this PR? It doesn't seem relevant to this thank you pages, and also seems like it's incorrect based on the markup on webinars anyway: 
```
<span class="rsWebinarTopic-item-link"><a href="#" class=""><%= linktext %></a></span>
```

Reply:
There are lint errors that I was trying to quiet. and yeah I now see the change while it quiets the lint error is not correct.